### PR TITLE
Improve Docker scripts

### PR DESCRIPTION
**Pull request by ManuelBilbao**
Fix and improve Docker initialization of the project

### CODE REVIEW

File: docker/scripts/node.entrypoint.sh [hunk 1]
```diff
@@ -44,3 +44,4 @@ PEER_LIST=$(IFS=,; echo "${PEERS[*]}")
 
 $CHAIN_BINARY config set config p2p.persistent_peers "$PEER_LIST" --skip-validate
 $CHAIN_BINARY config set app minimum-gas-prices "$MINIMUM_GAS_PRICES" --skip-validate
+$CHAIN_BINARY config set config rpc.laddr "tcp://0.0.0.0:26657" --skip-validate
```
Comment (JulianGCalderon) [inline docker/scripts/node.entrypoint.sh:47]: Minimum gas prices should be already set by app config

**Review by JulianGCalderon — CHANGES_REQUESTED** [review]

File: docker/scripts/node.entrypoint.sh (hunk 1 above)
Comment (ManuelBilbao) [reply docker/scripts/node.entrypoint.sh]: Done

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Review by fborello-lambda — DISMISSED** [review]

**Review by MauroToscano — APPROVED** [review]

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Review by tomyrd — APPROVED** [review]
LGTM

**Pull request merged**